compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -608,11 +608,7 @@ void addAMDAIEObjectFifoLoweringPasses(OpPassManager &passManager) {
   passManager.addPass(createCSEPass());
   passManager.addPass(createCanonicalizerPass());
 
-  passManager.addPass(createAMDAIEDmaLoopSubsumptionPass());
-  passManager.addPass(createCSEPass());
-  passManager.addPass(createCanonicalizerPass());
-
-  passManager.addPass(createAMDAIECombineStridedOpsPass());
+  passManager.addPass(createAMDAIEDmaCompositionPass());
```

Comment:
I was considering that as well, but I think this is not canonicalization as it doesn't just change the operation on itself, so that would be confusing.

Reply:
I am also expecting there will be more transformations being added to this new pass as well, like creating chains of DMA operations (`next_bd`).